Source/BLE.Client/BLE.Client.Maui/Platforms/Android/AndroidManifest.xml

```diff
@@ -14,5 +16,8 @@
 	<uses-permission android:name="android.permission.BLUETOOTH_PRIVILEGED" />
 	<uses-permission android:name="android.permission.BLUETOOTH_SCAN" android:usesPermissionFlags="neverForLocation" />
 	<uses-permission android:name="android.permission.CHANGE_NETWORK_STATE" />
-	<uses-permission android:name="android.permission.CONTROL_LOCATION_UPDATES" />
+	<uses-permission android:name="android.permission.CONTROL_LOCATION_UPDATES" />-->
+	<uses-feature android:name="android.hardware.bluetooth_le" android:required="true"/>
+	<uses-permission android:name="android.permission.BLUETOOTH_SCAN" android:usesPermissionFlags="neverForLocation" />
+	<uses-permission android:name="android.permission.BLUETOOTH_CONNECT" />
```

Comment:
I'm afraid you're breaking the permissions on older Android versions here: At least `BLUETOOTH` and `ACCESS_FINE_LOCATION` are needed on Android 11 and below. The other ones are probably ok to be removed.

Reply:
True.. I indeed started from my own use case scenario to test whether those "minimal" rights would cover my case.
I'll reinstate the other related permissions by checking the Android docs on them.